src/frontend/src/binder/expr/function.rs

```diff
@@ -953,6 +953,11 @@ impl Binder {
                         Ok(ExprImpl::literal_bool(false))
                 }))),
                 ("pg_tablespace_location", guard_by_len(1, raw_literal(ExprImpl::literal_null(DataType::Varchar)))),
+                // TODO: really implement them.
+                // https://www.postgresql.org/docs/9.5/functions-info.html#FUNCTIONS-INFO-COMMENT-TABLE
+                ("col_description", raw_literal(ExprImpl::literal_varchar("".to_string()))),
```

Comment:
Non-related to this PR, we could further rewrite the function to sub-query on system tables including `pg_description`.